src/Catel.MVVM/Catel.MVVM.Shared/Collections/Extensions/SuspensionContextExtensions.cs

```diff
@@ -89,16 +185,14 @@ public static bool IsMixedMode<T>(this SuspensionContext<T> suspensionContext)
         }
 
         /// <summary>
-        /// The create mixed bash event args list.
+        /// The create bash event args list.
         /// </summary>
         /// <param name="suspensionContext">The suspension context.</param>
+        /// <param name="suspensionMode">The suspension mode.</param>
         /// <typeparam name="T">The type of collection item.</typeparam>
-        /// <returns>The <see cref="ICollection{NotifyRangedCollectionChangedEventArgs}"/>.</returns>
-        private static ICollection<NotifyRangedCollectionChangedEventArgs> CreateMixedBashEventArgsList<T>(this SuspensionContext<T> suspensionContext)
+        /// <returns>The <see cref="IList{NotifyRangedCollectionChangedEventArgs}"/>.</returns>
+        private static IList<NotifyRangedCollectionChangedEventArgs> CreateBashEventArgsList<T>(this SuspensionContext<T> suspensionContext, SuspensionMode suspensionMode)
```

Comment:
If you want it public than please make the original function public but do not alter the behaviour.